machine_configuration/README.md

```diff
@@ -16,7 +16,7 @@ If the configuration cannot be set as the default setting, we recommend using so
 Any contribution in this area is welcome. 
 
 # Prerequisites 
-- Provisioned RHOCP 4.12 cluster. Follow steps [here](/README.md#provisioning-rhocp-cluster).
+- Provisioned RHOCP 4.13 cluster. Follow steps [here](/README.md#provisioning-rhocp-cluster).
```

Comment:
According to @vbedida79's suggestion, let us remove the OCP version here, and just say provisioned RHOCP cluster

Reply:
Sounds good